src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,57 @@
+@import "./utils/function";
+@import "./utils/mixin";
+@import "./utils/variables";
```

Comment:
You need this imports only in main.scss before your block imports.
Or only here, then remove them from main.scss